modal/_functions.py

```diff
@@ -136,8 +136,10 @@ async def create(
         function_call_invocation_type: "api_pb2.FunctionCallInvocationType.ValueType",
     ) -> "_Invocation":
         assert client.stub
+        stub = client.stub
```

Comment:
Eventually, this will be

```py
        stub = await ModalInputPlaneModal.create(client, function._input_plane_server_url)
```

This constructor is `O(# of gRPC methods)`; for now, there are only two gRPC methods. I'm assuming this is OK, but it would be good to get confirmation from others.

If this is too slow, we can add a `(StubType, server URL) -> Stub` cache, but personally I think it's not necessary.

Reply:
Probably fine, but something about it itches especially given that the Invocation itself will only use a handful of the rpc endpoints

Reply:
Yeah :/ We can split it into `ModalInputPlaneClient` and `ModalControlPlaneClient` but it seems undesirable to end up with a large number of stubs / services. We can always refactor / revisit this later.